core/primitives/src/account.rs

```diff
@@ -20,7 +20,14 @@ pub struct Account {
 
 impl Account {
     pub fn new(amount: Balance, code_hash: CryptoHash, storage_paid_at: BlockIndex) -> Self {
-        Account { amount, staked: 0, code_hash, storage_usage: 0, storage_paid_at }
+        let account_storage_overhead = 100;
+        Account {
```

Comment:
We should have this as a number in the config. See my comment below. And assign storage_usage upon executing action_create_account instead of doing it inside the constructor of Account. In this way we avoid having this number hardcoded in the primitives crate.